src/components/VariableCreateModal.vue

```diff
@@ -114,6 +119,9 @@
         showToast(localization.success.createVariable, 'success')
         emit('create', variable)
         internalValue.value = false
+        name.value = undefined
+        value.value = undefined
+        tags.value = []
```

Comment:
this is resetting the form state right? I'd extract this into a separate `reset` method. This is something I wish prefect-design could handle for us.